observ/scheduler.py

```diff
@@ -44,13 +48,27 @@ def register_qt(self):
         else:
             raise ImportError("Could not import QtCore")
 
-        self.timer = QtCore.QTimer()
-        self.timer.setSingleShot(True)
-        self.timer.timeout.connect(scheduler.flush)
-        # Set interval to 0 to trigger the timer as soon
-        # as possible (when Qt is done processing events)
-        self.timer.setInterval(0)
-        self.register_request_flush(self.timer.start)
+        try:
+            QtAsyncio = importlib.import_module(f"{qt}.QtAsyncio")
+            import asyncio
+
+            asyncio.set_event_loop_policy(QtAsyncio.QAsyncioEventLoopPolicy())
```

Comment:
My expectation is that users of observ (application developers) will set the Qt event loop policy in their application initialization code.

Observ just needs to a `register_asyncio()` function that users can then call. That makes observ compatible with any asyncio-compatible framework.

We can keep `register_qt()` around for backward compatibility.

Reply:
Makes a lot of sense!

Reply:
Fixed!